### DISCUSSION TITLE
Issues with Safari <= 13 ?

### DISCUSSION THREAD

**Discussion by t35h** [vercel/next.js#55309] [Help] [answered]
### Summary

A project I'm working on using 13.4 is no longer rendering the page on Safari 13 on MacOS High Sierra and Safari on iPhone 6. I noticed the NextJS website also no longer loads on these devices. Tested on colleagues old Macbook and through the BrowserStack.

Anyone else noticing this?

**Comment by icyJoseph** [comment]
Mmm most likely a polyfill missing or a feature that's not supported in those browsers. Got any logs?

**Reply by t35h** [reply]
`undefined is not an object (evaluating e.startTime)` is the error I saw in the console before BrowserStack closed the session because of the 1 minute free trial limitation.

**Reply by icyJoseph** [reply]
Oh yeah then I think it has to be with some timing API. I saw someone else with the same exact problem this week.

**Reply by icyJoseph** [reply, marked as answer]
Alright, looks like it is fixed in Canary:

- https://github.com/vercel/next.js/issues/55084#issuecomment-1712455398

So you probably should wait for a new version.

Or use Canary if you are ok with that.

**Reply by t35h** [reply]
Excellent. Thanks!
